samples/snippets/cloud-client/src/main/java/privateca/ActivateSubordinateCa.java

```diff
@@ -0,0 +1,132 @@
+import com.google.cloud.security.privateca.v1.CertificateAuthorityName;
+import com.google.cloud.security.privateca.v1.CertificateAuthorityServiceClient;
+import com.google.cloud.security.privateca.v1.SubordinateConfig;
+import com.google.longrunning.Operation;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+public class ActivateSubordinateCa {
+
+  public static void main(String[] args)
+      throws InterruptedException, ExecutionException, IOException {
+    // TODO(developer): Replace these variables before running the sample.
+
+    // location: For a list of locations, see:
+    // https://cloud.google.com/certificate-authority-service/docs/locations
+    // pool_Id: Set a unique id for the CA pool.
+    // subordinateCaName: The CA to be activated.
+    // pemCACertificate: The signed certificate, obtained by signing the CSR.
+    String project = "your-project-id";
+    String location = "ca-location";
+    String pool_Id = "ca-pool-id";
+    String subordinateCaName = "subordinate-certificate-authority-name";
+    String pemCACertificate =
+        "-----BEGIN CERTIFICATE-----\n" + "sample-pem-certificate\n" + "-----END CERTIFICATE-----";
+
+    // certificateAuthorityName: The name of the certificate authority which signed the CSR.
+    // If an external CA (CA not present in GCloud) was used for signing, then use the CA's
+    // issuerCertificateChain.
+    String certificateAuthorityName = "certificate-authority-name";
+
+    activateSubordinateCA(
+        project, location, pool_Id, certificateAuthorityName, subordinateCaName, pemCACertificate);
+  }
+
+  // Activate a subordinate CA.
+  // *Prerequisite*: Get the CSR of the subordinate CA signed by another CA. Pass in the signed
+  // certificate and (issuer CA's name or the issuer CA's Certificate chain).
+  // *Post*: After activating the subordinate CA, it should be enabled before issuing certificates.
+  public static void activateSubordinateCA(
+      String project,
+      String location,
+      String pool_Id,
+      String certificateAuthorityName,
+      String subordinateCaName,
+      String pemCACertificate)
+      throws ExecutionException, InterruptedException, IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the `certificateAuthorityServiceClient.close()` method on the client to safely
+    // clean up any remaining background resources.
+    try (CertificateAuthorityServiceClient certificateAuthorityServiceClient =
+        CertificateAuthorityServiceClient.create()) {
+      // Subordinate CA parent.
+      String subordinateCaParent =
+          CertificateAuthorityName.of(project, location, pool_Id, subordinateCaName).toString();
+
+      // Create the Activate CA Request.
```

Comment:
"Activate the CA"

Reply:
The below instance doesn't call the activate ca API, but merely constructs the request.
Hence, I have re-worded it to -  Construct the "Activate CA Request".